### PR TITLE
wgengine/netstack: make netstack MTU be 1280 also

### PR DESCRIPTION
**Pull request by bradfitz**
Updates #3878

### CODE REVIEW

**Review by raggi — APPROVED** [review]

File: wgengine/netstack/netstack.go [hunk 1]
```diff
@@ -131,7 +131,7 @@ type Impl struct {
 var handleSSH func(logger.Logf, *ipnlocal.LocalBackend, net.Conn) error
 
 const nicID = 1
-const mtu = 1500
+const mtu = 1280
```
[the remark this answers is not in the document]
Comment (bradfitz) [reply wgengine/netstack/netstack.go:134]: Yeah, I can make them share a constant somewhere.

But I'm more concerned that this change isn't sufficient. @danderson suggested the MSS was still not right after this.

[the remark this answers is not in the document]
Comment (raggi) [reply wgengine/netstack/netstack.go:134]: I'd like to see a pcap, I'm wondering if TFO is involved too?

[the remark this answers is not in the document]
Comment (bradfitz) [reply wgengine/netstack/netstack.go:134]: I doubt it, because ~nothing uses TFO.

**Review by crawshaw — APPROVED** [review]
I think it would be relatively easy to add a test to the end of tstest/integration/vms/vms_test.go, adding one more tsnet based client. A bit messy piling tests for everything in there, but it's already spun up normal clients on ubuntu so a tcp connection from a tsnet client to one should test this problem.

**Pull request merged**